aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -18,6 +18,22 @@ Tensor quantize_per_tensor_cpu(
   return quantizer->quantize(self);
 }
 
+std::vector<Tensor> quantize_per_tensor_list_cpu(
+    TensorList tensors,
+    Tensor scales,
+    Tensor zero_points,
+    ScalarType dtype) {
+  std::vector<Tensor> quantized_tensors;
+  for (auto i = 0; i < tensors.size(); ++i) {
+    quantized_tensors.push_back(quantize_per_tensor_cpu(
```

Comment:
should we call aten native function here?

Reply:
Either way works I think. That function calls `quantize_per_tensor_cpu`